src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -324,6 +330,17 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('password1', options_list=['--password1'], help='Flag indicating if first password should be deleted', action='store_true', required=False)
         c.argument('password2', options_list=['--password2'], help='Flag indicating if second password should be deleted.', action='store_true', required=False)
 
+    with self.argument_context('acr agentpool') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.argument('agent_pool_name', options_list=['--name', '-n'], help='The name of the agent pool.')
+        c.argument('count', type=int, help='The count of the agent pool.')
+        c.argument('tier', help='Sets the VM your agent pool will run on. Valid values are: S1(Standard_D2_v3), S2(Standard_D4_v3) or S3(Standard_D8_v3)')
```

Comment:
S1 (2 cpus)
S2 (4 cpus)
S3 (8 cpus)

Reply:
Sets the VM tier*

@northtyphoon , we should probably also mention memory to be thorough
S1 (2 cpus, 3 MB RAM)
S2 (4 cpus, 8 MB RAM)
S3 (8 cpus, 16 MB RAM)